wherescape/connectors/hubspot/process_data.py

```diff
@@ -113,18 +113,17 @@ def compare_names(source_names: list, destination_names: list):
     - known_destination_names (list) : list of property names that also exist as column names in the source table being used
     """
     known_destination_names = []
+    unknown_destination_names = []
 
     for name in source_names:
         if name not in destination_names and name != "record_id" and "dss_" not in name:
-            """
-            Only prints a warning if it doesn't have dss_ in it
-            """
-            logging.warning(
-                "source name: %s does not exist in the destination. Please check its existence and spelling"
-                % name
-            )
+            unknown_destination_names.append(name)
+        elif "dss_" in name:
```

Comment:
A comment here would be good I think; stating that these are generated columns for during the modeling process.